dev/io.openliberty.microprofile.telemetry.1.0.internal/resources/io/openliberty/microprofile/telemetry/internal/resources/MPTelemetry.nlsprops

```diff
@@ -29,3 +29,7 @@
 CWMOT5100.tracing.is.disabled=CWMOT5100I: The MicroProfile Telemetry Tracing feature is enabled but not configured to generate traces for the {0} application.
 CWMOT5100.tracing.is.disabled.explanation=The MicroProfile Telemetry Tracing feature needs to be configured to create traces and export them to a tracing system.
 CWMOT5100.tracing.is.disabled.useraction=To enable tracing, specify otel.sdk.disabled=false in any of the configuration sources accessible through MicroProfile Config. Further configuration might be required to allow the MicroProfile Telemetry Tracing feature to export traces to your tracing system.
+
+CEMOT5101.telemetry.error=CEMOT5101: An exception was thrown from MicroProfile Telemetry. See the assocaited exception: {0}.
+CEMOT5101.telemetry.error.explanation=An exception was thrown from within MicroProfile Telemetry and logged.
+CEMOT5101.telemetry.error.useraction=Examine the wrapped exception and perform any appropriate actions.
```

Comment:
@benjamin-confino @helyarp How about something like this?

This is similar to what we wrote for MP Reactive Messaging.

```suggestion
CEMOT5101.telemetry.error=CEMOT5101: An internal error occurred in MicroProfile Telemetry. The error is: {0}.
CEMOT5101.telemetry.error.explanation=An error has occurred that prevents normal operation of MicroProfile Telemetry.
CEMOT5101.telemetry.error.useraction=Review the server message.log file and FFDC logs to identify the problem.
```